.github/workflows/checks.yaml

```diff
@@ -24,11 +24,8 @@ jobs:
         with:
           channel: stable
 
-      - name: Install dependencies
-        run: flutter pub get
-
       - name: Analyze source code
-        run: flutter analyze --fatal-infos --fatal-warnings
+        run: flutter analyze --pub --fatal-infos --fatal-warnings
```

Comment:
Sadly `flutter analyze` does not fail on `public_member_api_docs`, but `dart analyze` does. Consider adding that here.

Reply:
As the majority of the code will be placed in `lib/src/` this will not matter anyway, since it won't be public. This suffices for now and can be changed once `ClipRegularPolygon` is introduced publicly.